### DISCUSSION TITLE
GitHub Copilot could not connect to server.

### DISCUSSION THREAD

**Discussion by hsnbozkurt** [community/community#10995] [Copilot Conversations] [answered]
[INFO] [auth] [2022-02-04T17:26:23.748Z] Invalid copilot token: invalid expires_at:1643995582 now:1643995583
[ERROR] [default] [2022-02-04T17:26:23.750Z] GitHub Copilot could not connect to server. Extension activation failed: "Failed to get copilot token"

**Comment by shlomiandadi** [comment]
Try turning the plug-in off and on.

**Comment by davecheney** [comment, marked as answer]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.
